addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -162,7 +162,7 @@ QUnit.test("rendering with PWA installation request", async (assert) => {
     assert.ok(
         target
             .querySelector(".o-mail-NotificationItem img")
-            .dataset.src.includes("/web/image?field=avatar_128&id=2&model=res.partner")
+            .dataset.src.includes("/web/image/res.partner/2/avatar_128")
```

Comment:
prefer full assert rather than includes (I want to see the unique key in all tests)